pyston/resource.py

```diff
@@ -710,7 +710,7 @@ def _get_form(self, fields=None, inst=None, data=None, files=None, initial={}):
             kwargs['files'] = files
 
         form_class = self._generate_form_class(inst, exclude)
-        return form_class(initial=initial, **kwargs)
+        return form_class(initial=initial or {}, **kwargs)
```

Comment:
v tom is None máš určitě pravdu, jinak jste mne už asi moc vycukali se svýma "zbytečnýma promněnnýma", proto jsem to naflákal rovnou na místo použití